auth/src/test/java/io/grpc/auth/GoogleAuthLibraryCallCredentialsTest.java

```diff
@@ -414,12 +403,18 @@ public void jwtAccessCredentialsInRequestMetadata() throws Exception {
             .build();
     GoogleAuthLibraryCallCredentials callCredentials =
         new GoogleAuthLibraryCallCredentials(credentials);
-    callCredentials.applyRequestMetadata(requestInfo, executor, applier);
+    callCredentials.applyRequestMetadata(new RequestInfoImpl("example.com:123"), executor, applier);
 
     verify(applier).apply(headersCaptor.capture());
     Metadata headers = headersCaptor.getValue();
-    assertArrayEquals(Iterables.toArray(expectedAuthMetadata, String.class),
-        Iterables.toArray(headers.getAll(AUTHORIZATION), String.class));
+    String token =
+        Iterables.getOnlyElement(headers.getAll(AUTHORIZATION)).substring("Bearer".length());
+    DecodedJWT decoded = JWT.decode(token);
+    assertEquals("test-private-key-id", decoded.getKeyId());
+    assertEquals("https://example.com:123/a.service",
+        Iterables.getOnlyElement(decoded.getAudience()));
+    assertEquals("test-email@example.com", decoded.getIssuer());
```

Comment:
Note that we don't really have to test these. That's more testing the credential library. The only thing we really need to check in the audience, since we provide part of that.

Reply:
The purpose of this test was to verify that our code correctly converts a `ServiceAccountCredentials` to `ServiceAccountJwtAccessCredentials`, with all provided information in `ServiceAccountCredentials` populated to the converted `ServiceAccountJwtAccessCredentials`. This is kind of the most "behavioral" way of testing that, without inspecting the implementation.

Reply:
Oh, true. That's fair.